src/ui/server/controllers/insurance/business/turnover/validation/rules/estimated-annual-turnover.ts

```diff
@@ -31,7 +31,7 @@ const estimatedAnnualTurnover = (formBody: RequestBody, errors: object) => {
 
   const allowNegativeValue = true;
 
-  return wholeNumberValidation(formBody, errors, errorMessage, FIELD_ID, allowNegativeValue);
+  return numberValidation({ formBody, errors, errorMessage, fieldId: FIELD_ID, allowNegativeValue });
```

Comment:
Can we make sure all the unit tests reflect what the function is returning:

`should return the result of numberValidation`